### PR TITLE
fix skarlso company association

### PR DESCRIPTION
**Pull request by Skarlso**
Hello.

I'm trying to fix my company affiliation, but I couldn't run any of the scripts locally. 

It complains with a bunch of error not finding the other scripts no matter where I'm running them from. :/

Help? :D

### CODE REVIEW

File: developers_affiliations2.txt [hunk 1]
```diff
@@ -1298,14 +1298,15 @@ Skandh15: skandhoff!gmail.com
 	Dayal Group from 2018-06-01 until 2018-07-01
 	Independent from 2018-07-01 until 2019-09-01
 	Optum Global Solutions from 2019-09-01
-Skarlso: 182850+Skarlso!users.noreply.github.com, Skarlso!users.noreply.github.com, gergely!weave.works
+Skarlso: 182850+Skarlso!users.noreply.github.com, Skarlso!users.noreply.github.com, gergely!weave.works gergely!kubermatic.com
```
Comment (craigbox) [inline developers_affiliations2.txt]:
```suggestion
Skarlso: 182850+Skarlso!users.noreply.github.com, Skarlso!users.noreply.github.com, gergely!weave.works, gergely!kubermatic.com
```

**Comment by lukaszgryglicki** [comment]
Hi, I'll TAL on Friday.

**Review by lukaszgryglicki — APPROVED** [review]
/lgtm - I'll check and eventually fix all manually. Thanks for reporting.

**Comment by Skarlso** [comment]
Yaaay thank you!🙏

**Pull request merged**